### PR TITLE
CI for concepts

### PR DESCRIPTION
**Pull request by wolf99**
I realised during #735  that tooling and CI does not yet cover concept exercises.
This PR should resolve that.

### CODE REVIEW

File: bin/check-unitybegin
Comment (wolf99) [inline bin/check-unitybegin:28]: Im not too sure if it is a good idea to leave this as a warning.
Maybe it should be an error.
It could mean that there is a exercises/concept directory but no concept exercises therein, but it could also mean that there is a concept exercise without the needed files within it.
But then - separation of concerns - should this script care about that?

File: bin/verify-unity-version
Comment (wolf99) [inline bin/verify-unity-version:47]: Same question about the error handling here.

**Comment by wolf99** [comment]
Please squash if merging!

File: bin/check-include-guards
Comment (ryanplusplus) [inline bin/check-include-guards:52]: How about making this an error? I can't think of a situation where we wouldn't want to make this obvious.

File: bin/check-unitybegin
Comment (ryanplusplus) [reply bin/check-unitybegin:28]: I'm in favor of making this an error. It's likely that it would get caught long before this, but I can't see any reason why we wouldn't take this opportunity to call attention to something unexpected.

**Comment by ryanplusplus** [comment]
This looks good. I'm happy to merge this once we've resolved the question about warnings and errors.

File: bin/check-include-guards
Comment (wolf99) [reply bin/check-include-guards:52]: The exercises/concept directory doesn't currently exist. So this would cause the CI to always fail until we get some concept exercises merged

We could maybe not pass that directory in yet, but then at least the first concept exercise will not have CI.
It's kind of a bootstrap problem.

Comment (ryanplusplus) [reply bin/check-include-guards:52]: My bad, I haven't been following the Concept exercises and didn't realize that none had been merged.

**Review by ryanplusplus — APPROVED** [review]

**Pull request merged**
